src/main/java/ua/realestate/management/service/UserService.java

```diff
@@ -14,16 +14,13 @@
 import java.util.Optional;
 
 @Slf4j
-@AllArgsConstructor
 @Service
 public class UserService implements UserDetailsService {
-    @Autowired
-    private UserRepository userRepository;
 
-//    private final UserRepository userRepository;
-//    public UserService(UserRepository userRepository){
-//        this.userRepository = userRepository;
-//    }
+    private final UserRepository userRepository;
+    public UserService(UserRepository userRepository){
```

Comment:
You still can put `@Authowired` above the constructor, it will highlight that constructor is expecting Spring Bean and not something else.

Reply:
fixed